### DISCUSSION TITLE
Vercel Dev and monorepos

### DISCUSSION THREAD

**Discussion by cosbgn** [vercel/vercel#7480] [Help] [answered]
Hello, I have a monorepo which looks like this:
/app
    - .vercel
    - package.json
/second-app
    - .vercel
    - package.json

On vercel I've setup the entry point of two separate project to be /app and /second-app. The deployment works perfectly, whenever I push to master both apps get build.

However `vc dev` doesn't work anymore. Whenever I do `cd app` and then run `vc dev` I get:

```
Vercel CLI 24.0.0
Error! /Users/c/Dev/proj/app/app doesn't exist
```
How can I set the entry point only for production? Or should I run `vc dev` from the main folder? In that case how can I chose which project it should run?

**Comment by cosbgn** [comment, marked as answer]
Solved by this: https://github.com/vercel/vercel/discussions/5294

**Comment by jean-smaug** [comment]
👋 Hi,

I created this package that aims to emulate Vercel behavior.
https://github.com/jean-smaug/vercel-dev-server

I'm using it in a monorepo with multiple Vercel projects.

Hope it will help.
